README-develop.md

```diff
@@ -32,6 +32,13 @@ docker-compose exec phpfpm /app/bin/console doctrine:migrations:migrate
 docker-compose exec phpfpm /app/bin/console fos:user:create --super-admin
 ```
 
+## Building js assets
+
+Watch for changes in js and css files and build development version:
+```
+yarn watch
+```
+
```

Comment:
The README-develop should probably tell developers that they need to run yarn install in every bundle, as the yarn watch command will report errors about missing dependencies.